scripts/llm/ptq.py

```diff
@@ -14,19 +14,19 @@
 
 import argparse
 
-from megatron.core.dist_checkpointing.validation import StrictHandling
-
-from nemo.collections.llm import quantization
+from nemo.collections import llm
+from nemo.collections.llm.modelopt import ExportConfig, QuantizationConfig
 
 
 def get_args():
     """Parses PTQ arguments."""
-
     parser = argparse.ArgumentParser(
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-        description="NeMo PTQ argument parser",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter, description="NeMo PTQ argument parser"
     )
     parser.add_argument("-nc", "--nemo_checkpoint", type=str, help="Source NeMo 2.0 checkpoint")
+    parser.add_argument(
+        "--tokenizer", type=str, help="Tokenizer to use. If not provided, model tokenizer will be used"
```

Comment:
Added the ability to override tokenizer in the model or support a case where nemo ckpt doesn't have a tokenizer by default. 
This is consistent with pruning and distillation.

Main use case is for llama ci model where model's vocab size is 50k but doesn't have tokenizer hence we use default nmt_tokenizer with 32k vocab size and we need to overwrite the tokenizer to gpt2 tokenizer.

I think the issue doesn't occur in ptq because we don't use tokenized CI dataset instead take HF text dataset (cnn_dailymail)